README.rst

```diff
@@ -40,23 +40,31 @@ these files should be located at ``/edx/app/edxapp/`` directory:
 Location of Transformed Logs
 ****************************
 
-Transformed event logs can be found in ``/edx/var/logs/tracking/tracking.log`` file.
+Transformed events are logged using **'logging.handlers.SysLogHandler'** with **'facility: local2'**.
+
+We need to create output files manually and set appropriate permissions for syslog user, please follow the steps below:
```

Comment:
what if we use `To do so, please follow the steps below`

Reply:
Done, please review again